packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -28,12 +28,22 @@ export const runQuery = (handler, query, excludes, pathPrefix) =>
       return page
     })
 
+    // siteUrl Validation
     if (!r.data.site.siteMetadata.siteUrl) {
       throw new Error(
         `SiteMetaData 'siteUrl' property is required. Check out the documentation to see a working example: https://www.gatsbyjs.org/packages/gatsby-plugin-sitemap/#how-to-use`
       )
     }
 
+    if (
```

Comment:
I think we should augment the above if instead of creating a whole new one.

Reply:
okay